renovate.json

```diff
@@ -26,6 +26,11 @@
       "groupName": "all BBC non-major dependencies",
       "updateTypes": ["minor", "patch"],
       "groupSlug": "psammead-minor-patch"
+    },
+    {
+      "packageNames": ["psammead"],
```

Comment:
Am I right in thinking from this example in the docs (https://docs.renovatebot.com/configuration-options/#matchpackagenames) that this will only apply this rule for a specific package, called `psammead` i.e. https://www.npmjs.com/package/psammead? I _think_ we'd want it for the pattern `@bbc/psammead-*` (not sure the correct way to apply a wildcard selector)?

Reply:
I believe you are correct, not sure about the wildcard is needed as it states in the example `The above will configure rangeStrategy to replace for any package starting with angular.`, so anything starting with `@bbc/psammead` should be impacted.

Reply:
Fixed in https://github.com/bbc/simorgh/pull/8808/commits/09eb93926613f793cbe4a965423ae7b69e82fce4

Reply:
Ah mega, nice one